server/schema/postgres.sql

```diff
@@ -81,6 +81,17 @@ CREATE TABLE IF NOT EXISTS events (
 	updated_at timestamp NOT NULL DEFAULT now()
 );
 
+CREATE TABLE IF NOT EXISTS invalid_events (
+	id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+	event_id uuid NOT NULL,
```

Comment:
Some nitpicking :) 
The `organization_id` should always be the second column in a table so that we can add an index on (organization_id, event_id). Then we can query faster all invalid events for an organization (what the list call does) or query all invalid events for a specific event via (organization_id, event_id).

Therefore I would add the following index below the create statement:

`CREATE INDEX IF NOT EXISTS organization_id_event_id ON invalid_events (organization_id, event_id);`